connector/servicegraphconnector/connector.go

```diff
@@ -395,15 +396,16 @@ func (p *serviceGraphConnector) onExpire(e *store.Edge) {
 }
 
 func (p *serviceGraphConnector) aggregateMetricsForEdge(e *store.Edge) {
-	metricKey := p.buildMetricKey(e.ClientService, e.ServerService, string(e.ConnectionType), e.Dimensions)
+	metricKey := p.buildMetricKey(e.ClientService, e.ServerService, string(e.ConnectionType), strconv.FormatBool(e.Failed), e.Dimensions)
 	dimensions := buildDimensions(e)
 
 	p.seriesMutex.Lock()
 	defer p.seriesMutex.Unlock()
 	p.updateSeries(metricKey, dimensions)
-	p.updateCountMetrics(metricKey)
 	if e.Failed {
 		p.updateErrorMetrics(metricKey)
+	} else {
+		p.updateCountMetrics(metricKey)
```

Comment:
This is major change to how the metrics work. `traces_service_graph_request_total` should count all occurrences, not only successful ones.

Reply:
@mapno  Thank you for your careful review, new commit has pushed, please review.